src/types/requestBody.ts

```diff
@@ -136,6 +143,8 @@ export interface Targets {
   index?: number;
   cache?: CacheSettings | string;
   targets?: Targets[];
+  /** This is used to determine if the request should be transformed to formData Example: Stability V2 */
+  transFormToFormData?: boolean;
```

Comment:
Can we rename this to `transformToFormData` (lower case f in transform)

Reply:
done